tools/run_tests/run_xds_tests.py

```diff
@@ -2804,13 +3120,15 @@ def __init__(self, compute, alpha_compute, project, project_num):
 
             try:
                 if not CLIENT_HOSTS:
-                    client_cmd_formatted = args.client_cmd.format(
-                        server_uri=server_uri,
+                    client_cmd_formatted_no_uri = args.client_cmd.format(
```

Comment:
Why `_no_uri`? Or did you mean `client_cmd_formatted_no_port` ?

Reply:
Because this string has every placeholder formatted except for `server_uri` placeholder, and yea basically I wanted to specify/omit the port in those server_uri to test default port scenario